pedant/decorators.py

```diff
@@ -197,7 +226,7 @@ def log_debug_render(*args, **kwargs):
 
     def log_render(*args, **kwargs):
         try:
-            variable_node_render(*args, **kwargs)
+            return variable_node_render(*args, **kwargs)
```

Comment:
This seemed to be an existing bug. Unlike the `debug` version of this function (just above), this one previously wasn't returning the actual result if there was no UnicodeDecodeError. This was probably just an oversight and wasn't caught by the tests because the tests only run the `debug` version on Django <= 1.8. Django 1.9 drops the `debug` versions of the template objects, which is how I noticed this bug.